docker/elasticsearch-setup/Dockerfile

```diff
@@ -3,10 +3,20 @@
 # Defining environment
 ARG APP_ENV=prod
 
+FROM golang:1.17-alpine3.15 AS binary
+ENV DOCKERIZE_VERSION 0.6.1
+WORKDIR /go/src/github.com/jwilder
+RUN apk --no-cache --update add openssl git tar curl \
+    && git clone https://github.com/jwilder/dockerize.git
+WORKDIR /go/src/github.com/jwilder/dockerize
+
+ENV GO111MODULE=on
+RUN go mod tidy
+RUN go install
```

Comment:
`go install github.com/jwilder/dockerize@latest`

might be simpler

Reply:
sorted on the branch, not too familiar with the suggestion from @szalai1 as I'm a go virgin, but for now it'll grab the specified tag

Reply:
@pedro93 OK, hopefully that should be it, moved directly to `go install github.com/jwilder/dockerize@...` as per @szalai1